api/types/image.go

```diff
@@ -12,6 +12,9 @@ type Image struct {
 	// NewName is the value used to replace the original name.
 	NewName string `json:"newName,omitempty" yaml:"newName,omitempty"`
 
+	// TagSuffix is the value used to suffix the original tag
+	TagSuffix string `json:"tagSuffix,omitempty" yaml:"tagSuffix,omitempty"`
+
 	// NewTag is the value used to replace the original tag.
```

Comment:
I think it's worth updating the comment on Digest and this one here to warn that if one of them is specified, TagSuffix will have no effect.

Or maybe add this info on the comment of TagSuffix itself. But in this case, I would update the comment of both NewTag and Digest to make this kind of information available in consistent places.

Reply:
I have added comments based on the second suggestion given